src/stylesheets/_resource_view_more_link.scss

```diff
@@ -0,0 +1,37 @@
+.resource-view-more-link {
+    position: relative;
+    cursor: pointer;
+    height: 4.4rem;
+    width: 4.4rem;
+    text-align: center;
+    display: inline-block;
+    border-radius: 9999px;
+    background-color: transparent;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    z-index: 99;
```

Comment:
This could be more specific.

Reply:
It's the same as we use for the context menu to ensure it's on top of other content in the tile.

Reply:
They both could be changed to something more sensible.